cmd/cycloid/creds/create.go

```diff
@@ -48,6 +48,7 @@ func NewCreateCommand() *cobra.Command {
 
 	WithPersistentFlagDescription(cmd)
 	common.RequiredPersistentFlag(WithPersistentFlagName, cmd)
+	common.WithPersistentFlagCan(cmd)
```

Comment:
don't forget to add --canonical example to the create command example list

Reply:
resolve after chat discussion